### DISCUSSION TITLE
What happened to minor breaks in 2023

### DISCUSSION THREAD

**Discussion by ygk2zhang** [has2k1/plotnine#698] [Q&A]
Hi,

I would like to know how to enable minor_breaks. I use Plotnine on GoogleColabs
Last year I could run:
------
from plotnine import *
import plotnine as plotnine
plotnine.options.figure_size=(7, 7)

(ggplot(pmf,aes(x='x_coord'))
+ geom_line(aes( y='predicted', color ='"blue"', linetype='"solid"'), size=1.5)+theme_classic(base_size=22)
+ geom_line(aes( y='y_coord', color ='"blue"', linetype='"dashdot"'), size=1)
+ geom_point(aes( y='predicted', color='"red"', shape='"o"'), size=0)+theme(legend_position=(0.77,0.85))
+ geom_point(aes( y='y_coord', color='"red"', shape='""'), size=3)+labs(x='Distance, Å', y='PMF, kcal/mol')+scale_x_continuous(minor_breaks=2)
+ scale_colour_manual(name=" ", labels=( 'Mesoscale', 'Atomistic'), values = ( "black","red"))
+scale_linetype_manual(name=" ", labels=( 'Mesoscale', 'Atomistic'), values =( 'solid',''))
+ scale_shape_manual(name=" ", labels=('Mesoscale', 'Atomistic'), values = ("","o"))
+ theme(axis_title_y=element_text(margin={'t': 0, 'b': 0, 'l': 0, 'r':-3}), axis_ticks_direction='in',axis_title_x=element_text(margin={'t':-2, 'b': 0, 'l': 0, 'r':0}) ))
-----
![image](https://github.com/has2k1/plotnine/assets/72882562/2dd6e597-8809-45fb-b45d-7718dc78ac28)

**Now if I rerun the same job on GoogleColabs with plotnine 0.10.0 , I don't get the minor breaks** I have plotnine 0.8 on my laptop and still do not get the minor breaks 

![image](https://github.com/has2k1/plotnine/assets/72882562/258a40fc-3ff4-4b3f-b64e-24bf470a7895)

The question is how to do I get get the minor breaks back?

**Comment by has2k1** [comment]
You can turn on the minor breaks with

```python
+ theme(axis_ticks=element_line())         # turn on major & minor breaks
+ theme(axis_ticks_minor=element_line())   # turn on minor breaks
+ theme(axis_ticks_minor_x=element_line()) # turn on minor breaks for the x axis
+ theme(axis_ticks_minor_y=element_line()) # turn on minor breaks for the y axis
```
